lib/api/model/model.dart

```diff
@@ -301,7 +301,8 @@ class Subscription {
 
   final int streamPostPolicy; // TODO enum
   // final bool? isAnnouncementOnly; // deprecated; ignore
-  final String emailAddress;
+
+  // final String emailAddress; // TODO(server-8) present before Server 7.5 (zulip/zulip@432001656)
```

Comment:
Because it's no longer in the list of properties of this object in the API docs (it's mentioned only in a "Changes" note at the top of the object type), we don't even need this tombstone ­— can just delete mention of the field entirely.